coins/testnet/btcp.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "btcprivate_testnet",
+    "symbol": "bttt",
```

Comment:
I don't know if this is the official symbol for the testnet btcp. If it's not, would you mind changing it to btcpt?